contribs/gmf/less/desktop.less

```diff
@@ -229,3 +229,172 @@ main {
     border-radius: @app-margin;
     background: @brand-primary;
 }
+
+
+/**
+ * GMF DrawFeature directive
+ */
+gmf-drawfeature h2 {
+  display: none;
+}
+
+.gmf-eol {
+  clear: both;
+}
+
+.gmf-drawfeature-featurelistctn {
+  border-top: 1px solid #333;
+  margin: 10px 0 0 0;
+  padding: 10px 0 0 0;
+}
+
+.gmf-drawfeature-featurelist {
+  margin: 10px 0 0 0;
+  padding: 0;
+  list-style: none;
+
+  li {
+    &:hover {
+      background-color: #ADD8E6;
+      cursor: pointer;
+    }
+  }
+}
+
+
+/**
+ * NGEO DrawFeature directive & map tooltips
+ */
+.ngeo-drawfeature-actionbuttons {
+  float: right;
+  position: relative;
+}
+
+.ol-viewport {
+  .tooltip {
+    position: relative;
+    background: rgba(0, 0, 0, 0.5);
+    border-radius: 4px;
+    color: white;
+    padding: 4px 8px;
+    opacity: 0.7;
+    white-space: nowrap;
+  }
+  .tooltip-measure {
+    opacity: 1;
+    font-weight: bold;
+  }
+  .tooltip-static {
+    display: none;
+  }
+  .tooltip-measure:before,
+  .tooltip-static:before {
+    border-top: 6px solid rgba(0, 0, 0, 0.5);
+    border-right: 6px solid transparent;
+    border-left: 6px solid transparent;
+    content: "";
+    position: absolute;
+    bottom: -6px;
+    margin-left: -7px;
+    left: 50%;
+  }
+  .tooltip-static:before {
+    border-top-color: #ffcc33;
+  }
+}
+
+
+/**
+ * GMF FeatureStyle directive
+ */
+gmf-featurestyle {
+  display: block;
+  margin: 10px 0 0 0;
+
+  .form-horizontal {
+    .control-label {
+      padding-top: 0;
+      text-align: left;
+    }
+  }
+
+  input[type=range] {
+    background-color: transparent;
+    padding: 6px 12px;
+  }
+
+  .gmf-featurestyle-name {
+    font-weight: bold;
+    font-size: 16pt;
+
+    &:not(:focus) {
+      background-color: transparent;
+      border: none;
+      box-shadow: none;
+      -webkit-box-shadow: none;
```

Comment:
Don't use prefixed rules, the [auto prefixer plugin](https://www.npmjs.com/package/less-plugin-autoprefix) already take care of that